internal/storage/storageutil/object_attrs_test.go

```diff
@@ -0,0 +1,102 @@
+package storageutil
+
+import (
+	"crypto/md5"
+	"testing"
+	"time"
+
+	"cloud.google.com/go/storage"
+	. "github.com/jacobsa/ogletest"
+	storagev1 "google.golang.org/api/storage/v1"
+)
+
+const TestBucketName string = "gcsfuse-default-bucket"
+const TestObjectName string = "gcsfuse/default.txt"
+const Test string = "test"
+
+func TestObjectAttrsToBucketObject(t *testing.T) { RunTests(t) }
```

Comment:
Please rename this  to TestObjectAttrs. It should be named after class name and not method name.

Reply:
fixed